Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.10.8-slim
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```

Comment:
Ensure that `main.py` is correctly located in the `app` directory to ensure the `CMD` instruction works properly.